c10/core/CPUAllocator.cpp

```diff
@@ -37,28 +37,66 @@ void memset_junk(void* data, size_t num) {
   }
 }
 
+void* alloc(size_t nbytes) {
```

Comment:
nit: I don't think it's wise to take up such a "prime" name `c10::alloc` for aligned CPU allocation. It's not something most users really need from us, and we might need the name for something else at some later point in time.